server/menus/tests.py

```diff
@@ -11,22 +13,74 @@ def setUp(self):
         self.test_user = User.objects.create(username="Foxy",
                                              first_name="Foxy")
         self.chef = Chef.objects.create(user=self.test_user)
-
+        self.menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
+        self.item = Item.objects.create(menu=self.menu, title="kibbles", size="the whole bowl",
+                                   price=13.33, description="food for foxy")
     def test_menu_creation(self):
         '''
         Makes sure menu is properly created and linked to chef.
         '''
 
-        # Tests creation of menu and connection to user
-        self.assertFalse(Menu.objects.filter(chef=self.chef).exists())
-        menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
-        self.assertTrue(Menu.objects.filter(chef=self.chef).exists())
-        self.assertEqual(self.chef, menu.chef)
-        self.assertEqual(True, menu.delivery)
+        # # Tests creation of menu and connection to user
+        # self.assertFalse(Menu.objects.filter(chef=self.chef).exists())
+        # menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
+        # self.assertTrue(Menu.objects.filter(chef=self.chef).exists())
```

Comment:
we can still use this test i think - but we can change the filter in line 27 to filter on title instead of menu (just in case there are other menus for that chef in the database already). so we assert on Menu.objects.filter(title='foxy food').exists() and then we can still test the link between menu and item in the rest of the test. it's not TOTALLY necessary, but no reason to throw it out. it just covers our bases even more. you can cut everything before line 27 though and start the test there!